R/read_data.R

```diff
@@ -0,0 +1,8 @@
+## Task 2
+
+read_data <- function(url, destfile) {
+  download.file (url = url, destfile = destfile)
+  
+  read_xlsx(file = destfile)
+}
+read_data(url = "https://github.com/OxfordIHTM/teaching_datasets/raw/refs/heads/main/cyclones.xlsx", destfile = "R/read_data")
```

Comment:
Since this is the functions R file/s, we don't put the implementation of the function here, so I would remove this line from here.